students/nartowska_karina/lesson_04_unit_testing/test_for_fibonacci_numbers.py

```diff
@@ -0,0 +1,42 @@
+import unittest
+
+
+def fib(n):
```

Comment:
Lines from 4 to 21 are not needed - the aim of this task is to test only fib function:) You can just import fib function from the fibonacci_numbers module from lesson_03_functions. To do it:
- mark your folder (nartowska_karina) as src folder (go to PyCharm-> right click on the karina_nartowska folder -> Mark Directory as -> choose source root
- Add empty __init__.py file in the lesson_03_functions to treat this directory as containing packages (have a look at https://docs.python.org/3/tutorial/modules.html#packages)
- now in the test_for_fibonacci_numbers,py you should have the possibility to import function you would like to test (from lesson_03_functions.fibonacci_numbers import fib)

Reply:
the proper name of the empty file you should add "__init__.py". Don't forget to push this file on GitHub as well